### DISCUSSION TITLE
During the dpgen process, step 04 will interrupt and report an error("run_tasks for model devi should not be empty! Please check your files.")，At the same time, the files required for subsequent tasks were not generated in the 01.model_devi folder

### DISCUSSION THREAD

**Discussion by XuFanffei** [deepmodeling/dpgen#1377] [Q&A]
DeePMD-kit Version
deepmd-kit—2.2.1

TensorFlow Version
2.10.0

How did you download the software?
Offline packages

Input Files, Running Commands, Error Log, etc.
run_param.json
{
"type_map": [
"Li","P","S","Cl"
],
"mass_map": [
6.9,30.97,32,35
],
"init_data_prefix": "./",
"init_data_sys": [
"data_init/md-222-0.95","data_init/md-222-1","data_init/md-222-1.05",
"data_init/md-unit-0.95","data_init/md-unit-1","data_init/md-unit-1.05"

],
"sys_format":"abacus/stru",
"sys_configs_prefix": "./",
"sys_configs": [
    [
	"../autotest/ab_relax/scale-1.00/eos_00/task.0000*/STRU"
    ]
],
"_comment": " that's all ",
"numb_models": 4,
"default_training_param": {
    "model": {
        "type_map": [
            "Li","P","S","Cl"
        ],
        "descriptor": {
            "type": "se_e2_a",
            "sel": [
                36,8,28,8
            ],
            "rcut_smth": 0.5,
            "rcut": 6.0,
            "neuron": [
                25,
                50,
                100
            ],
            "resnet_dt": false,
            "axis_neuron": 16,
            "seed": 1
        },
        "fitting_net": {
            "neuron": [
                240,
                240,
                240
            ],
            "resnet_dt": true,
            "seed": 1
        }
    },
    "learning_rate": {
        "type": "exp",
        "start_lr": 0.001,
        "stop_lr":3.51e-8,
        "decay_steps": 2500
    },
    "loss": {
        "start_pref_e": 0.02,
        "limit_pref_e": 2,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
        "start_pref_v": 0.01,
        "limit_pref_v": 1.0
    },
    "training": {
        "stop_batch": 400000,
        "disp_file": "lcurve.out",
        "disp_freq": 2000,
        "numb_test": 4,
        "save_freq": 20000,
        "save_ckpt": "model.ckpt",
        "disp_training": true,
        "time_training": true,
        "profiling": false,
        "profiling_file": "timeline.json",
        "_comment": "that's all"
    }
},
"model_devi_dt": 0.002,
"model_devi_skip": 0,
"model_devi_f_trust_lo": 0.15,
"model_devi_f_trust_hi": 0.35,
"model_devi_clean_traj": false,
"model_devi_jobs": [
    { "sys_idx": [0], "temps": [1200,700,500], "_press": [1.0], "trj_freq": 10,"nsteps": 20000, "ensemble": "nvt", "_idx": "00"},
    { "sys_idx": [0], "temps": [1200,700,500], "_press": [1.0], "trj_freq": 10,"nsteps": 50000, "ensemble": "nvt", "_idx": "01"},
    { "sys_idx": [0], "temps": [1200,700,500], "_press": [1.0], "trj_freq": 10,"nsteps": 80000, "ensemble": "nvt", "_idx": "02"},
    { "sys_idx": [0], "temps": [1200,700,500], "_press": [1.0], "trj_freq": 10,"nsteps": 80000, "ensemble": "nvt", "_idx": "03"},
    { "sys_idx": [0], "temps": [1200,700,500], "_press": [1.0], "trj_freq": 10,"nsteps": 80000, "ensemble": "nvt", "_idx": "04"}
],
"fp_style": "abacus",
"shuffle_poscar": false,
"fp_task_max": 800,
"fp_task_min": 10,
"fp_pp_path": "./",
"fp_pp_files": ["Li_ONCV_PBE-1.0.upf","P_ONCV_PBE-1.0.upf",
                  "S_ONCV_PBE-1.0.upf","Cl_ONCV_PBE-1.0.upf"],
"fp_orb_files": ["Li_gga_9au_100Ry_4s1p.orb","P_gga_8au_100Ry_2s2p1d.orb",
                 "S_gga_8au_100Ry_2s2p1d.orb","Cl_gga_8au_100Ry_2s2p1d.orb"],
"k_points": [3,3,3,0,0,0],
"user_fp_params":{
    "calculation": "scf",
	"ntype": 4,
"symmetry": 0,
"basis_type": "lcao",
"ecutwfc": 60,
	"scf_thr": 1e-8,
	"scf_nmax": 100,
"smearing_method": "gauss",
	"smearing_sigma": 0.02,
	"mixing_type": "pulay",
	"mixing_beta": 0.4,
	"cal_force": 1,
    "cal_stress":   1
}

**Comment by SchrodingersCattt** [comment]
I am encountering the same problem. May I ask if your problem is solved now and how?

**Reply by HABERLO** [reply]
Hello, may I ask about "Run_Tuskersformoder_Devischordnott Bjönputi!" Is the problem solved?

**Reply by wenkeji** [reply]
I am also encountering the same problem. May I ask if your problem is solved now and how?

**Comment by njzjz** [comment]
Please check whether the file defined in `sys_configs` exists.

**Comment by darkgezi** [comment]
for my case, after  changing the name of configs from POSCAR[6-10] to POSCAR[1-5],the problem solved
